src/nsis.mk

```diff
@@ -18,25 +18,27 @@ define $(PKG)_UPDATE
 endef
 
 define $(PKG)_SCONS_OPTS
-    XGCC_W32_PREFIX='$(TARGET)-' \
+    CCACHE_DISABLE=1 \
+    PATH='$(PREFIX)/bin:$(PATH)' \
     PREFIX='$(PREFIX)/$(TARGET)' \
-    `[ -d /usr/local/include ] && echo APPEND_CPPPATH=/usr/local/include` \
-    `[ -d /usr/local/lib ]     && echo APPEND_LIBPATH=/usr/local/lib` \
-    $(if $(findstring x86_64-w64-mingw32,$(TARGET)),\
-        SKIPPLUGINS='System' TARGET_ARCH=amd64) \
+    XGCC_W32_PREFIX='$(TARGET)-' \
+    CPPPATH_APPEND="['#$(PREFIX)/$(TARGET)/include'$(shell [ -d /usr/local/include ] && echo , \'#/usr/local/include\')]" \
+    LIBPATH_APPEND="['#$(PREFIX)/$(TARGET)/lib'$(shell [ -d /usr/local/lib ] && echo , \'#/usr/local/lib\')]" \
```

Comment:
- why rename `APPEND_CPPPATH` -> `CPPPATH_APPEND`:
```
% grep -r CPPPATH_APPEND gits/nsis-3.05-src
# no result
% grep -r APPEND_CPPPATH gits/nsis-3.05-src
gits/nsis-3.05-src/SConstruct:opts.Add(PathVariable('APPEND_CPPPATH', 'Additional paths to search for include files', None))
gits/nsis-3.05-src/SConstruct:defenv.Append(CPPPATH = Split('$APPEND_CPPPATH'))
```
  - what are the hashes for? I'm used to switching between shell and make, but I can't parse those lines.

Reply:
> * why rename `APPEND_CPPPATH` -> `CPPPATH_APPEND`:

Because I'm dumb and/or had slept too little.

> ```
> % grep -r CPPPATH_APPEND gits/nsis-3.05-src
> # no result
> % grep -r APPEND_CPPPATH gits/nsis-3.05-src
> gits/nsis-3.05-src/SConstruct:opts.Add(PathVariable('APPEND_CPPPATH', 'Additional paths to search for include files', None))
> gits/nsis-3.05-src/SConstruct:defenv.Append(CPPPATH = Split('$APPEND_CPPPATH'))
> ```
> 
> * what are the hashes for? I'm used to switching between shell and make, but I can't parse those lines.

They're probably not strictly needed but I added them when nothing else was working (can't remember why); anyway, the `#` tells scons that it should use the top-level directory as reference; although I don't really think that's relevant for absolute paths.

Reply:
Given that `scons` wasn't seeing them at all, can you remove those lines? 

I'm happy if you pop them out to a comment since it's not clear they are relevant any longer.